code/main/edge/locationHintManager.brs

```diff
@@ -15,12 +15,23 @@
 
 function _adb_LocationHintManager() as object
 
-    return {
+    locationHintManager = {
         _DEFAULT_LOCATION_HINT_TTL_SEC: 1800,
         _EDGE_NETWORK_SCOPE: "edgenetwork",
 
         _locationHint: invalid,
-        _timer: invalid,
+        _initTSInMillis&: _adb_InternalConstants().TIMESTAMP.INVALID_VALUE,
+        _expiryTSInMillis&: _adb_InternalConstants().TIMESTAMP.INVALID_VALUE,
+
+        _init: function() as void
+            locationHintObject = m._loadLocationHintObject()
+            if not _adb_isEmptyOrInvalidMap(locationHintObject)
```

Comment:
nit: _saveLocationHintObject() persists these variables directly. You can keep both functions consistent and have _loadLocationHintObject() just update these variables.